x-pack/solutions/security/plugins/elastic_assistant/server/lib/attack_discovery/schedules/register_schedule/definition.ts

```diff
@@ -35,6 +36,7 @@ export const getAttackDiscoveryScheduleType = ({
     category: DEFAULT_APP_CATEGORIES.security.id,
     producer: 'siem',
     solution: 'security',
+    priority: TaskPriority.NormalLongRunning,
```

Comment:
Thanks for updating scheduled Attack discoveries to use this new (lower) priority!